cmd/earthly/subcmd/account_cmds.go

```diff
@@ -602,12 +602,14 @@ func (a *Account) actionLogin(cliCtx *cli.Context) error {
 	}
 
 	loggedInEmail, authType, _, err := cloudClient.WhoAmI(cliCtx.Context)
-	if err == nil {
+
+	if err == nil && authType == cloud.AuthMethodCachedJWT {
```

Comment:
I wonder if this also fixes the bug/behavior change where in order to login to user Y, while already being logged into user X, forces an explicit `earthly account logout`

Reply:
Good question. I can look into that. We should also verify whether we want to keep this skipping logic.